src/Features/Core/Portable/QuickInfo/Presentation/QuickInfoContentBuilderContext.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.CodeAnalysis.Classification;
+using Microsoft.CodeAnalysis.Formatting;
+
+namespace Microsoft.CodeAnalysis.QuickInfo.Presentation;
+
+internal sealed class QuickInfoContentBuilderContext(
```

Comment:
📝 This was essentially moved from https://github.com/dotnet/roslyn/blob/main/src/EditorFeatures/Core/IntelliSense/QuickInfo/IntellisenseQuickInfoBuilderContext.cs.